src/contracts/reader/AllowListStorageReader.sol

```diff
@@ -1,23 +1,22 @@
 // SPDX-License-Identifier: LGPL-3.0-or-later
 pragma solidity ^0.7.6;
 
-import "@gnosis.pm/util-contracts/contracts/StorageAccessible.sol";
-import "@openzeppelin/contracts/utils/EnumerableSet.sol";
-import "../GPv2AllowListAuthentication.sol";
-
 /// @title Gnosis Protocol v2 Allow List Storage Reader
 /// @author Gnosis Developers
 contract AllowListStorageReader {
-    using EnumerableSet for EnumerableSet.AddressSet;
-
-    address private _owner;
-    EnumerableSet.AddressSet private solvers;
-
-    function getSolverAt(uint256 index) external view returns (address) {
-        return solvers.at(index);
-    }
+    address private manager;
+    mapping(address => bool) private solvers;
 
-    function numSolvers() external view returns (uint256) {
-        return solvers.length();
```

Comment:
I like that a new view method was added, but I don't see why the old ones need to be removed?

Reply:
Since we just have a simple mapping now, there is no way to get the number of solvers on-chain anymore.

Reply:
Right, and I suppose we also can't access solvers by index anymore either..

Reply:
Correct.